### PR TITLE
Add component testing

### PR DESCRIPTION
**Pull request by sjmiller609**
Hey @andriisoldatenko I'm wondering if you are interested in the idea of running component testing on commander by running it alongside Kind https://github.com/kubernetes-sigs/kind , and using python "behave" do to behavior testing.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+version: 2
```
Comment (andriisoldatenko) [inline .circleci/config.yml]: why not drone?

Comment (sjmiller609) [reply .circleci/config.yml]: It's because it was hard to run KinD in docker. It's easier on a VM

**Comment by sjmiller609** [comment]
The reason I'm doing this is because I want to push more integration/live testing to the "left" i.e. closer to the developers. If this works well, I might do similar (deploy using Kind) for our helm repo.

File: Gopkg.lock [hunk 2]
```diff
@@ -2,87 +2,112 @@
 
 
 [[projects]]
+  digest = "1:180876db3ec295bb9f0babec5ca926fe9f2036b747b7c5bfcd13b333023e7cfd"
```
Comment (sjmiller609) [inline Gopkg.lock]: this file was changed on accident, I will revert it

**Comment by sjmiller609** [comment]
Greg had reservations about using python since it's go

**Comment by sjmiller609** [comment]
I spoke with Andrii in PM and he agrees that it's a good choice to stick to Go for the component testing. I'll switch that over, then re-open.

File: .circleci/config.yml (hunk 1 above)
Comment (andriisoldatenko) [reply .circleci/config.yml]: hm... it's possible to run it in drone, similar to minikube:
```
root@97d5b9ffb602:/# minikube start --vm-driver=none --kubernetes-version=v1.15.0
```

or pls review this repo https://github.com/whindes/alpine-minikube

**Comment by sjmiller609** [comment]
I'm hoping we can reconsider using this style of testing for commander @schnie @andriisoldatenko.

**Comment by sjmiller609** [comment]
it would allow us to write functional tests cases for helm upgrades

**Comment by schnie** [comment]
Are we reworking this in a new branch now?

**Pull request closed without merging**